src/QMCWaveFunctions/Fermion/MultiSlaterDeterminantFast.cpp

```diff
@@ -550,6 +550,7 @@ void MultiSlaterDeterminantFast::evaluateDerivatives(ParticleSet& P,
                                                      std::vector<ValueType>& dlogpsi,
                                                      std::vector<ValueType>& dhpsioverpsi)
 {
+  this->evaluateDerivativesWF(P, optvars, dlogpsi);
```

Comment:
Remove `this->`. You are calling the local one. Please remove all similar use.

Reply:
Removed.